Tasks/RunDistributedTests/RunDistributedTests.ps1

```diff
@@ -24,6 +24,95 @@ Function CmdletHasMember($memberName) {
     return $cmdletParameter
 }
 
+#To parse the Overriding Parameters string provided by user
+Function GetPropertiesMapping {
```

Comment:
Should be Get-PropertiesMapping

Reply:
modified

---
In reply to: [137519575](https://github.com/Microsoft/vsts-tasks/pull/5246#discussion_r137519575) [](ancestors = 137519575)